content/en/blog/todo-ossna19.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "TODO at Open Source Summit North America"
+author: todogroup
+date: 2019-08-15
+---
+
+At [Open Source Summit North America](https://events.linuxfoundation.org/events/open-source-summit-north-america-2019/), the TODO Group will lead a track featuring a variety of talks dedicated to open source program management:
+
+Listed below are the talks from the TODO track at OSSNA: 
+
+Wednesday, August 21
+ * [Metrics - When One Size Doesn’t Fit All](https://ossna19.sched.com/event/PUTM/metrics-when-one-size-doesnt-fit-all-carsten-jacobsen-uber-technologies-inc-manrique-lopez-bitergia) - Carsten Jacobsen, Uber Technologies Inc. & Manrique Lopez, Bitergia
+ * [The Developer Goodie Bag](https://ossna19.sched.com/event/PUUo/the-developer-goodie-bag-carsten-jacobsen-uber-technologies-inc) - Carsten Jacobsen, Uber Technologies Inc.
+ * [Shifting Incentives in Open Source Participation](https://ossna19.sched.com/event/PUTV/shifting-incentives-in-open-source-participation-craig-northway-qualcomm-technologies-inc) - Craig Northway, Qualcomm Technologies Inc
+ * [Navigating the Interface Between Open and Closed Source Software](https://ossna19.sched.com/event/PUVC/navigating-the-interface-between-open-and-closed-source-software-craig-peters-lachlan-evenson-microsoft) - Craig Peters & Lachlan Evenson, Microsoft
+ * [Your Company Cares About Open Source Sustainability. But Are You Measuring and Encouraging Upstream Contributions?](https://ossna19.sched.com/event/PUVm/your-company-cares-about-open-source-sustainability-but-are-you-measuring-and-encouraging-upstream-contributions-danielle-gellis-indeed) - Danielle Gellis, Indeed
+ * [Open Source Collaboration and Companies: Finding the Right Balance](https://ossna19.sched.com/event/PUTw/open-source-collaboration-and-companies-finding-the-right-balance-dawn-m-foster-pivotal) - Dawn M. Foster, Pivotal
+ 
+Thursday, August 22
```

Comment:
I’d say make the dates headings so they stand out a bit more

Reply:
I feel like I caused a dumb somewhere, because when I check the blog, it does not show up and it should.